### PR TITLE
Project details

### PR DESCRIPTION
**Pull request by jr-cast**
updates:

* Extra JS functionalities were added to detail cards using JS events
* Mobile version of details card
* Desktop version of details card
* JS object implemented to update specific information depending on card/project 
      - links
      -description data
      -buttons
      -anchors
      -headers, etc

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -625,6 +625,121 @@ form textarea:focus {
   text-align: center;
 }
 
+@media all and (max-width: 768px) {
```
Comment (nicupop729) [inline style.css:628]: For mobile, unfortunately, the container which holds the pop-up is bigger than the screen. 

- [ ] Please redesign the pop-up for mobile versions in order to fit in between the edges of the screen. At the moment, the container is so large that there is no access to the close module button:
image

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @jr-cast & @Trictonicmp, 

Good job implementing the pop-up!
There are some issues that you still need to work on to go to the next project but you are almost there!




### Required Changes ♻️

In order to have a dynamic page and to respect the DRY principle, it is recommended that we make use of JavaScipt to render information that is easy to store in data structures. Imagine you have more than 20 projects that you want to display. It will make your HTML file very big and with a lot of repetitive containers. 
For this milestone, one of the requirements was to delete the cards from the HTML file and to render them dynamically from JavaScript:
image

- [ ] In order to pass this milestone, please remove the hardcoded cards from HTML and make use of JavaScript to display them. (you should loop thru the array that you created, take the information you need from the array and using DOM manipulation, display each card on the UI);

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jr-cast** [comment]
image

image

File: app.js [hunk 2]
```diff
@@ -36,4 +36,85 @@ function redirect() {
...
+    detImg.alt = projectsStore[id]['det-img-alt'];
+    document.getElementById('detailsHeader').innerHTML = projectsStore[id]['det-header'];
+    const detUl = document.getElementById('det-list-cont');
+    detUl.innerHTML = '';
+    for (let i = 0; i < projectsStore[id]['det-list-cont'].length; i += 1) {
+      const detLi = document.createElement('li');
+      detLi.innerHTML = projectsStore[id]['det-list-cont'][i];
+      detUl.appendChild(detLi);
+    }
+    document.getElementById('det-description').innerHTML = projectsStore[id]['det-description'];
```
[the remark this answers is not in the document]
Comment (sja-thedude) [reply app.js:117]: Good job making the changes as suggested by the reviewer.

File: style.css (hunk 1 above)
Comment (sja-thedude) [reply style.css:649]: Good job making the changes as suggested by the reviewer.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @jr-cast, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**